experiment/measurer/coverage_utils.py

```diff
@@ -293,3 +295,42 @@ def extract_covered_regions_from_summary_json(summary_json_file):
     except Exception:  # pylint: disable=broad-except
         logger.error('Coverage summary json file defective or missing.')
     return covered_regions
+
+
+def extract_segments_and_functions_from_summary_json(  # pylint: disable=too-many-arguments
+        cov_summary_file, benchmark, fuzzer, trial_id, time, segment_df,
+        function_df, recorded_segments):
```

Comment:
Can you try to keep parameter names across functions as much as possible?
`time` -> `this_time`
`trial_id` -> `trial_num`
`segment_df` -> `segment_coverage`
`function_df` -> `function_coverage`